test/unit/line-ripple/index.test.tsx

```diff
@@ -90,40 +95,42 @@ test('#adapter.hasClass returns true if exists in classList', () => {
   const classList = new Set();
   classList.add('test-color-class');
   wrapper.setState({classList});
-  const hasClass = wrapper
-    .instance()
-    .foundation_.adapter_.hasClass('test-color-class');
+  const hasClass = getAdapter(wrapper.instance().foundation_).hasClass('test-color-class');
   assert.isTrue(hasClass);
 });
 
 test('#adapter.setStyle updates style', () => {
   const wrapper = shallow<LineRipple>(<LineRipple />);
-  wrapper.instance().foundation_.adapter_.setStyle('color', 'blue');
+  getAdapter(wrapper.instance().foundation_).setStyle('color', 'blue');
   const style = coerceForTesting<React.CSSProperties>(wrapper.state().style);
   assert.equal(style.color, 'blue');
 });
 
 test('onTransitionEnd calls the #foundation.handleTransitionEnd', () => {
   const wrapper = shallow<LineRipple>(<LineRipple />);
-  wrapper.instance().foundation_.handleTransitionEnd = td.func();
-  const event = {test: '123'};
+  wrapper.instance().foundation_.handleTransitionEnd = td.func<(env: TransitionEvent) => null>();
+  const event: React.TransitionEvent = {
+    nativeEvent: {
+      test: '123',
+    },
+  } as any;
```

Comment:
please change to unknown to follow best practices

https://github.com/material-components/material-components-web/blob/master/docs/code/best_practices.md#any-vs-unknown-vs-

Reply:
Please use `coerceForTesting` https://github.com/material-components/material-components-web/blob/master/docs/code/best_practices.md#any-vs-unknown-vs-